eevee/metrics/classification.py

```diff
@@ -3,6 +3,8 @@
 import pandas as pd
 from sklearn.metrics import classification_report, precision_recall_fscore_support
 
+TRUE = "intent_x"
```

Comment:
Hard to find this variable at top level and figure out what it is. Maybe omething like 'TRUE_LABEL' would be better